src/index.js

```diff
@@ -23,6 +24,8 @@ function xml(o, name) {
   let value = o;
   if (typeof o === 'object') {
     value = Object.keys(o).map(k => xml(o[k], k)).join('\n');
+  } else if (typeof o === 'string') {
+    value = escape(o);
```

Comment:
well, that's not what I meant. for XML response, I think the `CDATA` is ok. but for the json response, it should be as-is. otherwise a json client would need to unescape the values again.

Reply:
For JSON, the escaping will be done by nodejs, because we return on object literal. For XML, all strings are escaped, but the escaping happens in the `xml` function, not in the `report` function.